martin/src/pg/scripts/query_available_tables.sql

```diff
@@ -50,6 +52,33 @@ WITH
                     geometry_columns.f_table_name = sic.table_name AND
                     geometry_columns.f_geometry_column = sic.column_name
         GROUP BY 1, 2, 3, 4, 5, 6),
+    --
```

Comment:
is geography metadata here substantially different from geometry to require a separate section? not a biggie, and not a blocker, but if possible - we should have just one consolidated sub-table here

Reply:
There are a few substantials differences here: geography columns are defined in the `geography_columns` table, and the name of the column containing the name of the spatial column is `f_geography_column`. I agree that the duplication here is not ideal, but I couldn't think of a better alternative to cover both use cases.

Perhaps one thing we could do is move the `JOIN`s to the `UNION`'d table, at the cost of adding yet another "with query" block? Not sure which of the two would be more preferable.